phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionObserver.java

```diff
@@ -1146,6 +1154,45 @@ private void setMutationAttributes(Mutation m, byte[] uuidValue) {
             m.setDurability(Durability.SKIP_WAL);
         }
 
+        private Delete generateDeleteMarkers(List<Cell> row) {
+            Set<ColumnReference> allColumns = indexMaintainer.getAllColumns();
+            if (row.size() == allColumns.size() + 1) {
```

Comment:
Can we have a comment here why we are returning null when we have 1 more row cell then allColumns?

Reply:
I will add a comment line. The reason we return is that we have all the columns for the index table plus the empty column.